internal/pkg/image/driver/imagedriver.go

```diff
@@ -245,17 +252,22 @@ func (d *fuseappsDriver) Mount(params *image.MountParams, mfunc image.MountFunc)
 		targetFd, _ := strconv.Atoi(path.Base(params.Source))
 		cmd.ExtraFiles[0] = os.NewFile(uintptr(targetFd), params.Source)
 	}
-	cmd.SysProcAttr = &syscall.SysProcAttr{
-		AmbientCaps: []uintptr{
-			uintptr(capabilities.Map["CAP_SYS_ADMIN"].Value),
-			// Needed for nsenter
-			//  https://stackoverflow.com/a/69724124/10457761
-			uintptr(capabilities.Map["CAP_SYS_PTRACE"].Value),
-		},
+
+	// when using gocryptfs for build step, we should not use SysProcAttr
+	if !(params.Filesystem == "gocryptfs" && params.SysProcOmit) {
```

Comment:
This could only check the params.SysProcOmit flag, it doesn't also have to check for gocryptfs. 

I like the name DontElevatePrivs better.